h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -15,6 +15,7 @@
 public class SQLManager {
 
   private final static String TEMP_TABLE_NAME = "table_for_h2o_import";
+  private static final String MAX_USR_CONNECTIONS_KEY = H2O.OptArgs.SYSTEM_PROP_PREFIX + "sql.connections.max";
```

Comment:
please use the same order of modifiers - i looks strange this way (I would change the others in this class)